Octokit.Tests.Integration/Clients/DeploymentsClientTests.cs

```diff
@@ -17,33 +19,38 @@ public DeploymentsClientTests()
 
         _deploymentsClient = github.Repository.Deployment;
         _context = github.CreateRepositoryContext("public-repo").Result;
+        _commits = new List<Commit>();
 
-        var blob = new NewBlob
+        for (int i = 0; i < 6; i++)
```

Comment:
Do you mean, I should move creation of these additional commits from test constructor to each ApiOption test method?